src/components/TranslateDialog.vue

```diff
@@ -0,0 +1,194 @@
+<!--
+  - @copyright Copyright (c) 2023 Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @author Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<NcModal @close="$emit('close')">
+		<div class="translate-dialog">
+			<h2> {{ t('spreed', 'Translate message') }} </h2>
+			<label for="from">
+				{{ t('spreed', 'From') }}
+			</label>
+			<NcSelect v-model="selectedFrom"
+				input-id="from"
+				:aria-label="t('spreed', 'From:')"
+				:placeholder="t('spreed', 'From:')"
+				:options="availableFroms"
+				:clearable="false"
+				track-by="from"
```

Comment:
track-by does not exist in NcSelect? At least it did not for me yesterday

Reply:
It worked for NcMultiSelect before, my bad

Reply:
Yeah, but NcMultiselect is deprecated because it's not accessible. So going with NcSelect is correct, you just need to map the "track-by" atttribute in your data to the id field

Reply:
Suggested changes for Select options, see below